### PR TITLE
Our motives

### PR DESCRIPTION
**Pull request by urvashi-code1255**
## Issue that this pull request solves

 Closes: #213 

## Proposed changes
Add the content provided for motive page.

### Brief description of what is fixed or changed
Added the content and the relevent icons in the motive section of home page.

## Types of changes

_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update (Documentation content changed)
- [ ] Other (please describe): 

## Checklist

_Put an `x` in the boxes that apply_

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] My changes does not break the current system and it passes all the current test cases.

## Screenshots

Please attach the screenshots of the changes made in case of change in user interface
Capture

## Other information

Any other information that is important to this pull request

### CODE REVIEW

File: frontend/src/components/home/motive/motive.js [hunk 1]
```diff
@@ -3,145 +3,140 @@ import './motive.css';
...
+            <div
+              className="single-features d-flex mt-30 wow fadeIn"
+              data-wow-duration="1s"
+              data-wow-delay="0.5s"
+            >
+              <div className="features-icon">
+                <i className="fab fa-github"></i>
+              </div>
+              <div className="features-content text-left">
+                <h4 className="features-title"><a href="/#">Get Inspired</a></h4>
```
Comment (tejan-singh) [inline frontend/src/components/home/motive/motive.js:84]: You can remove this dummy data card. We are keeping only the updated cards

File: frontend/src/components/home/motive/motive.js [hunk 2]
```diff
@@ -3,145 +3,140 @@ import './motive.css';
...
+              className="single-features d-flex mt-30 wow fadeIn"
+              data-wow-duration="1s"
+              data-wow-delay="1s"
+            >
+              <div className="features-icon">
+                <i className="fab fa-github"></i>
+              </div>
+              <div className="features-content text-left">
+                <h4 className="features-title">
+                  <a href="/#">Networking Session</a>
```
Comment (tejan-singh) [inline frontend/src/components/home/motive/motive.js:104]: You can remove this dummy data card. We are keeping only the updated cards

File: frontend/src/components/home/motive/motive.js [hunk 3]
```diff
@@ -3,145 +3,140 @@ import './motive.css';
...
+            <div
+              className="single-features d-flex mt-30 wow fadeIn"
+              data-wow-duration="1s"
+              data-wow-delay="1.5s"
+            >
+              <div className="features-icon">
+                <i className="fab fa-github"></i>
+              </div>
+              <div className="features-content text-left">
+                <h4 className="features-title"><a href="/#">Meet New Faces</a></h4>
```
Comment (tejan-singh) [inline frontend/src/components/home/motive/motive.js:124]: You can remove this dummy data card. We are keeping only the updated cards

**Review by tejan-singh — CHANGES_REQUESTED** [review]

File: frontend/src/components/auth/login/login.css [hunk 4]
```diff
@@ -89,22 +99,25 @@
   color: #777777;
 }
 
-.login-btn {
+.fa-eye-slash, .fa-eye{
+  cursor: pointer;
+}
+
+/* .login-btn {
```
Comment (mohit355) [inline frontend/src/components/auth/login/login.css:106]: Remove the  extra code

File: frontend/src/components/contact/contact_us.css [hunk 5]
```diff
@@ -89,11 +89,11 @@
   color: #777777;
 }
 
-.contact-input input:focus,
+/* .contact-input input:focus,
```
Comment (mohit355) [inline frontend/src/components/contact/contact_us.css:92]: Remove these extra commented code

File: frontend/src/components/auth/login/login.css [hunk 6]
```diff
@@ -77,10 +86,14 @@
 }
 
 .login-input input:focus {
-  border-color: #1863ff !important ;
+  /* border-color: #1863ff !important ; */
```
Comment (Surajbokde) [inline frontend/src/components/auth/login/login.css]: remove this comment

**Review by Surajbokde — APPROVED** [review]

**Comment by urvashi-code1255** [comment]
@tejan-singh @mohit355 @Surajbokde I did all the changes. Please check..

**Review by Surajbokde — APPROVED** [review]

**Review by tejan-singh — APPROVED** [review]

**Review by mohit355 — APPROVED** [review]

**Comment by urvashi-code1255** [comment]
@Kajol-Kumari @jackfrost13 I am Dwoc And Swoc participant .

**Pull request merged**